package.json

```diff
@@ -126,6 +126,7 @@
     "jss-nested": "^1.0.1",
     "lodash": "^3.7.0",
     "lost": "^6.7.1",
+    "moment": "^2.11.2",
```

Comment:
I don't think this package is all the translations. For that, you have to include moment-locales I believe.